cvat/schema.yml

```diff
@@ -5403,6 +5511,9 @@ paths:
   /api/tasks/{id}/status:
     get:
       operationId: tasks_retrieve_status
+      description: 'This method is deprecated and will be removed in version 2.14.0.
```

Comment:
If this is deprecated, how about updating the SDK so that it doesn't use it anymore?